src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -566,6 +566,38 @@ void java_bytecode_instrumentt::operator()(exprt &expr)
   instrument_code(expr);
 }
 
+/// Instruments the code attached to `symbol` with
+/// runtime exceptions or corresponding assertions.
+/// Exceptions are thrown when the `throw_runtime_exceptions` flag is set.
+/// Otherwise, assertions are emitted.
+/// \par parameters: `symbol_table`: global symbol table (may gain exception type
```

Comment:
Plus this doxy comment appears managed - can be fixed by removing the back ticks and re running it through the converter